### PR TITLE
Adds open/close sound to blastdoors and shutters

### PR DESCRIPTION
**Pull request by nero1024**
Makes those silent blast doors and shutters play a sound. Also first PR so let me know if something iffy is going on.

:cl: Nero1024
soundadd: blast doors and shutters will now play a sound when they open and close.
/:cl:

The blast doors were awfully quiet. You would expect them to make noises when they open and close.

### CODE REVIEW

**Comment by WJohn** [comment]
Ooooh hell no a massive "CHUNK" noise playing a dozen times from all nearby blast doors on say, the bridge, would be fucking incredibly loud and ear-rape.

**Comment by nero1024** [comment]
@WJohn I've considered that knowing how sound seems to works in this game, but from my experience in the hours of testing before making the PR I've yet to experience any ear rape.

**Comment by optimumtact** [comment]
well did anyone notice

**Comment by nero1024** [comment]
@optimumtact Notice what? The test merge candidate tag?

**Comment by optimumtact** [comment]
it was test merged

File: code/game/machinery/doors/poddoor.dm [hunk 1]
```diff
@@ -49,8 +51,10 @@
 	switch(animation)
 		if("opening")
 			flick("opening", src)
+			playsound(src.loc, blastdoorOpen, 30, 1)
 		if("closing")
 			flick("closing", src)
+			playsound(src.loc, blastdoorClose, 30, 1)
```
Comment (Cyberboss) [inline code/game/machinery/doors/poddoor.dm]: remove .loc

File: code/game/machinery/doors/poddoor.dm [hunk 2]
```diff
@@ -49,8 +51,10 @@
 	switch(animation)
 		if("opening")
 			flick("opening", src)
+			playsound(src.loc, blastdoorOpen, 30, 1)
```
Comment (Cyberboss) [inline code/game/machinery/doors/poddoor.dm]: remove .loc

File: code/game/machinery/doors/poddoor.dm [hunk 3]
```diff
@@ -12,6 +12,8 @@
 	armor = list(melee = 50, bullet = 100, laser = 100, energy = 100, bomb = 50, bio = 100, rad = 100, fire = 100, acid = 70)
 	resistance_flags = FIRE_PROOF
 	damage_deflection = 70
+	var/blastdoorOpen = 'sound/machines/blastdoor.ogg'
+	var/blastdoorClose = 'sound/machines/blastdoor.ogg'
```
Comment (Cyberboss) [inline code/game/machinery/doors/poddoor.dm]: These don't need to be vars

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**